src/test/resources/outputs/trino/basics/exclude.sql

```diff
@@ -201,16 +201,19 @@ WHERE "t1"."a" AND "t2"."a";
-SELECT CAST(ROW("t"."flds"."a", CAST(ROW("t"."flds"."c"."field_y") AS ROW(field_y VARCHAR))) AS ROW(a ROW(field_x INTEGER, field_y VARCHAR), c ROW(field_y VARCHAR))) AS "flds", "t"."foo" AS "foo" FROM "default"."EXCLUDE_T_NULLABLE" AS "t";
+SELECT CAST(ROW("t"."flds"."a", CAST(ROW("t"."flds"."c"."field_y") AS ROW("field_y" VARCHAR))) AS ROW("a" ROW("field_x" INTEGER, "field_y" VARCHAR), "c" ROW("field_y" VARCHAR))) AS "flds", "t"."foo" AS "foo" FROM "default"."EXCLUDE_T_NULLABLE" AS "t";
 
 --#[exclude-50]
-SELECT transform("t"."a", ___coll_wildcard___ -> CAST(ROW(___coll_wildcard___."field_y", ___coll_wildcard___."field_z", ___coll_wildcard___."nested_list") AS ROW(field_y VARCHAR, field_z VARCHAR, nested_list ARRAY<INTEGER>))) AS "a", "t"."foo" AS "foo" FROM "default"."EXCLUDE_T_NESTED_LIST" AS "t";
+SELECT transform("t"."a", ___coll_wildcard___ -> CAST(ROW(___coll_wildcard___."field_y", ___coll_wildcard___."field_z", ___coll_wildcard___."nested_list") AS ROW("field_y" VARCHAR, "field_z" VARCHAR, "nested_list" ARRAY<INTEGER>))) AS "a", "t"."foo" AS "foo" FROM "default"."EXCLUDE_T_NESTED_LIST" AS "t";
 
 --#[exclude-51]
-SELECT transform("t"."a", ___coll_wildcard___ -> CAST(ROW(___coll_wildcard___."field_x", ___coll_wildcard___."field_z", ___coll_wildcard___."nested_list") AS ROW(field_x INTEGER, field_z VARCHAR, nested_list ARRAY<INTEGER>))) AS "a", "t"."foo" AS "foo" FROM "default"."EXCLUDE_T_NESTED_LIST" AS "t";
+SELECT transform("t"."a", ___coll_wildcard___ -> CAST(ROW(___coll_wildcard___."field_x", ___coll_wildcard___."field_z", ___coll_wildcard___."nested_list") AS ROW("field_x" INTEGER, "field_z" VARCHAR, "nested_list" ARRAY<INTEGER>))) AS "a", "t"."foo" AS "foo" FROM "default"."EXCLUDE_T_NESTED_LIST" AS "t";
 
 --#[exclude-52]
-SELECT transform("t"."a", ___coll_wildcard___ -> CAST(ROW(___coll_wildcard___."field_x", ___coll_wildcard___."field_y", ___coll_wildcard___."nested_list") AS ROW(field_x INTEGER, field_y VARCHAR, nested_list ARRAY<INTEGER>))) AS "a", "t"."foo" AS "foo" FROM "default"."EXCLUDE_T_NESTED_LIST" AS "t";
+SELECT transform("t"."a", ___coll_wildcard___ -> CAST(ROW(___coll_wildcard___."field_x", ___coll_wildcard___."field_y", ___coll_wildcard___."nested_list") AS ROW("field_x" INTEGER, "field_y" VARCHAR, "nested_list" ARRAY<INTEGER>))) AS "a", "t"."foo" AS "foo" FROM "default"."EXCLUDE_T_NESTED_LIST" AS "t";
 
 --#[exclude-53]
-SELECT transform("t"."a", ___coll_wildcard___ -> CAST(ROW(___coll_wildcard___."field_x", ___coll_wildcard___."field_y", ___coll_wildcard___."field_z") AS ROW(field_x INTEGER, field_y VARCHAR, field_z VARCHAR))) AS "a", "t"."foo" AS "foo" FROM "default"."EXCLUDE_T_NESTED_LIST" AS "t";
+SELECT transform("t"."a", ___coll_wildcard___ -> CAST(ROW(___coll_wildcard___."field_x", ___coll_wildcard___."field_y", ___coll_wildcard___."field_z") AS ROW("field_x" INTEGER, "field_y" VARCHAR, "field_z" VARCHAR))) AS "a", "t"."foo" AS "foo" FROM "default"."EXCLUDE_T_NESTED_LIST" AS "t";
+
+--#[exclude-54]
```

Comment:
(self-review): new test that would previously fail in Trino. Other tests just wrap double-quotes around the ROW field names.